src/components/InventoryGroups/utils/api.js

```diff
@@ -23,7 +23,16 @@ export const getGroups = (
     parameters.length > 0 ? '?' + parameters : ''
   }`;
 
-  return instance.get(path);
+  try {
+    return await instance.get(path);
+  } catch (e) {
+    if (e?.status === 403) {
+      return {
+        total: 0,
```

Comment:
Why return this here? 

Would this not just silence the issue?

I'm still for failing the whole batch when one request fails, regardless of what error it is. And the catching of those errors should be higher up and if one of the errors was a 403, it should direct the user to a login page.

Reply:
In this case the 403 is a legit response from the api, the first request checks if user has permissions to groups. In case he has not, then no more requests are made and you get the total as 0

Reply:
How can you know it is the/a first request?